lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -103,7 +104,8 @@ data SignedVote = SignedVote
 {-# ANN module ("HLint: ignore Use <$>" :: String) #-}
 
 getBlockHeader :: Get BlockHeader
-getBlockHeader =  (fromIntegral <$> getWord16be) >>= \s -> isolate s $ do
+getBlockHeader = label "getBlockHeader"
+    $ (fromIntegral <$> getWord16be) >>= \s -> isolate s $ do
     version <- getWord16be
```

Comment:
Just nitpicking, but shouldn't this line (and the lines following it) be further indented by one block? Like this:

```hs
getBlockHeader = label "getBlockHeader" $
    (fromIntegral <$> getWord16be) >>= \s -> isolate s $ do
        version <- getWord16be
        contentSize <- getWord32be
        slotEpoch <- fromIntegral <$> getWord32be
        slotId <- fromIntegral <$> getWord32be
        ...
```

Reply:
Should it?

I view this as
```haskell
function = auxiliarySetup $ do
   important
   things
```

Reply:
I would argue that this function is slightly different from the others though, as the first line is wrapped, which introduces an extra level of indentation.

Clearly, both of the variants below are valid Haskell syntax:

**Variant 1:**
```hs
getBlockHeader = label "getBlockHeader" $
    (fromIntegral <$> getWord16be) >>= \s -> isolate s $ do
    version <- getWord16be
    contentSize <- getWord32be
    slotEpoch <- fromIntegral <$> getWord32be
    slotId <- fromIntegral <$> getWord32be
```
**Variant 2:**
```hs
getBlockHeader = label "getBlockHeader" $
    (fromIntegral <$> getWord16be) >>= \s -> isolate s $ do
        version <- getWord16be
        contentSize <- getWord32be
        slotEpoch <- fromIntegral <$> getWord32be
        slotId <- fromIntegral <$> getWord32be
```
In my opinion, the extra indentation of **variant 2** makes the code more readable, as it gives a visual indication of exactly where the `do` block starts, i.e., that it is separate from the line starting `(fromIntegral <$> getWord16be) >>= \s -> isolate s`.

What do you think?

Reply:
Yes — I suppose 😄

I guess the extra indentation shattered my ability to ignore the `(fromIntegral [...])` line, but that might have been 1) just because I was used to it, and 2) bad because it's obfuscating.

So I think this makes perfect sense. Thanks! 👍